docs/developer/getting-started/concepts.md

```diff
@@ -0,0 +1,63 @@
+# Concepts
+
+This section is intended to provide an outline of background concepts behind Rancher, Kubernetes and containers. It is intended to be used as a starting by new hires who may not come from a DevOps background and may not have experience with Kubernetes or containerization.
+
+It can be hard to explain Rancher in layman's terms because it's hard to understand a technology if you don't understand the problem it is intended to solve. Rancher was originally designed to help enterprises solve problems that come with running many containers in production at scale, so if you haven't used containers in production, it can help to learn about containers and Kubernetes before learning about Rancher itself.
+
+The following learning path is a suggested guide for which order to learn Rancher concepts. Each concept in this order builds on the one before. If you are studying Kubernetes online, it is best to focus on materials that begin with explaining containers and container images and build on that knowledge from there.
+
+# Official Courses
+
+SUSE provides official training on Rancher and K3s. The courses are on the [SUSE community website.](https://community.suse.com/all-courses) However, because Rancher is built on top of Kubernetes and Kubernetes manages containers, it is ideal to have at least some familiarity with containers and Kubernetes before diving into Rancher.
+
+# Learning path
+
+## 1. Terminology
+
+It may be helpful to start with the [onboarding glossary for new hires](https://confluence.suse.com/display/EN/Onboarding+and+New+Hire+Glossary) and/or the [official Kubernetes glossary.](https://kubernetes.io/docs/reference/glossary/?fundamental=true)
```

Comment:
The confluence document is not available externally - I'd remove it for now.

Reply:
removed